source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst

```diff
@@ -47,8 +55,11 @@ For more complete documentation on using MinIO with Prometheus, see :ref:`How to
          :class: copyable
 
          mc [GLOBALFLAGS] admin prometheus metrics  \
-                                           ALIAS    \
-                                           [TYPE]
+                                           ALIAS                                           \
+                                           [TYPE]                                          \
+                                           [--api_version v3]                              \
+                                           [TYPE --bucket <bucket name> --api_version v3]
```

Comment:
```suggestion
                                           [--bucket <bucket name>]
```

You can specify that `--bucket` requires `--api_version` in the flag description.